Makefile

```diff
@@ -370,14 +375,28 @@ lint_fix:
 	git diff $(MASTER_BRANCH) --name-only --diff-filter AMR | grep -v "^docs/" | xargs isort && \
 	echo "--- isort inplace fixing done. ---;"
 
-test_no_lint:
-	@. $(VENVNAME)/bin/activate; \
+test_no_lint: _warn_misssing_repos_if_using_actor
+	@echo "============= snactor sanity-check ipu ===============" 2>&1
+	. $(VENVNAME)/bin/activate; \
 	snactor repo find --path repos/; \
-	for dir in repos/system_upgrade/*/; do \
-		echo "Running sanity-check in $$dir"; \
-		(cd $$dir && snactor workflow sanity-check ipu); \
-	done; \
-	$(_PYTHON_VENV) -m pytest $(REPORT_ARG) $(TEST_PATHS) $(LIBRARY_PATH) $(PYTEST_ARGS)
+	for dir in $$(echo $(REPOSITORIES) | tr "," " "); do \
+		echo "Running sanity-check in $(_SYSUPG_REPOS)/$$dir"; \
+		(cd $(_SYSUPG_REPOS)/$$dir && snactor workflow sanity-check ipu); \
+	done
+
+	@echo "==================== unit tests ======================" 2>&1;
+# the below commands need to be one shell invocation for the early exit to work;
+# note: need to store the paths into separate var as it here as it's lazily
+# evaluated on each use :), using ?= for the assignment does not help for
+# some reason
```

Comment:
These have to be unindented, otherwise they are treated as bash not make comments and get echoed by make :/. Another solution is to prefix all of them with the '@'.

Reply:
btw, if you use spaces instead of tabs, it will be also processed as comment. but rather do it this way, as otherwise it could happen that someone by mistake add spaces instead of tabs to a real code/recipe.

Reply:
I didn't know that, but it makes sense. I will leave it as it is.